setup.cfg

```diff
@@ -0,0 +1,6 @@
+[options.packages.find]
+where = src
```

Comment:
Don't need to include this in config file

Reply:
Removed now :+1: